market_maker_keeper/imtoken_utils.py

```diff
@@ -98,7 +100,7 @@ def _get_price_response(self, amount):
             }
 
         query_pair = f"{quote}/{base}"
```

Comment:
why is the `query_pair` have this construct `ETH/DAI` instead of `ETH-DAI`

Reply:
it is the way IMToken expects pairs, see https://docs.token.im/tokenlon-mmsk/en/#pairs-interface